prifi.sh

```diff
@@ -12,7 +12,7 @@
 
 # variables that you might change often
 
-dbg_lvl=3                       # 1=less verbose, 3=more verbose. goes up to 5, but then prints the SDA's message (network framework)
+dbg_lvl=1                       # 1=less verbose, 3=more verbose. goes up to 5, but then prints the SDA's message (network framework)
```

Comment:
Please keep log-level 3 on what you commit (so if you use 1, just dont `git add` this file)